release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
```

Comment:
@jldohmann 
Please change entry as follows: 
```
==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using an IBM Secure Execution-ready QCOW2 boot image.

To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition config. Addtionally, you must use LUKS encryption for root file system integrity.

For more information, see [insert xref to "Installing {op-system} using IBM Secure Execution"]
```
name of new file to reference to will be: `ibm-z-secure-execution.adoc`
Once I have the PR I'll let you know.